### ISSUE TITLE
[Bug] Switch button works wrong (translation mistake)

### ISSUE THREAD

**Issue by eruxa** [YTVanced/Vanced#423]
**Bug description**
Button that hides shorts shelf is reversed in russian language. It says "shorts are enabled" but in fact shorts shelf is disabled. And in the opposite direction, when it says "disabled" actually it's enabled. 

**Variant**
Non-Root

**Vanced version**
16.02.35 - Build - 01.47.00

**Android version**
Android 9.0 - Pie

**Device**
Samsung Galaxy S8 Plus

**Steps to Reproduce**
Steps to reproduce the error:
1. Open the app
2. Click on icon at the top right corner
3. Go to Settings - Vanced Settings - Ad Settings 
4. Switch the "Shorts shelf" button

**Expected behavior**
Disable shorts when switch says it's disabled

![изображение](https://user-images.githubusercontent.com/44018221/106888162-3b227000-66f7-11eb-9024-846cd4177ffd.png)

_Attach images/videos if possible. These can be helpful in solving the issue._

**Comment by Vendicated** [comment]
Duplicate of #410

**Comment by Vendicated** [comment]
My bad, brainfarted

**Comment by Vendicated** [comment]
You could fix the translation yourself: <https://translate.vancedapp.com/project/vanced/ru>

**Comment by milindgoel15** [comment]
No, google introduced shorts later on. Hence, vanced has yet to implement anything that declutters yt which may/may not looked upon.
